src/frontends/ir/tests/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+set(TARGET_NAME ov_ir_frontend_tests)
+
+ov_add_test_target(
+        NAME ${TARGET_NAME}
+        ROOT ${CMAKE_CURRENT_SOURCE_DIR}
+        DEPENDENCIES
+            openvino_ir_frontend
+        LINK_LIBRARIES
+            gtest
+            gtest_main
+            openvino::runtime::dev
+            commonTestUtils
+        INCLUDES
+            "${CMAKE_CURRENT_SOURCE_DIR}/../include"
+        ADD_CLANG_FORMAT
+        LABELS
+            IR_FRONTEND
```

Comment:
About files: src/frontends/ir/tests/partial_shape_deserialization.cpp and src/frontends/ir/tests/rt_info_deserialization.cpp

Please review these files they contain a lot of legacy API, in new tests please use only new API.